### PR TITLE
chore: remove all dangerous deleteAll and removeAll mutations

### PR DESCRIPTION
**Pull request by shreddedbacon**
# General Checklist

- [ ] Affected Issues have been mentioned in the Closing issues section
- [ ] Documentation has been written/updated
- [ ] PR title is ready for inclusion in changelog

# Database Migrations

- [ ] If your PR contains a database migation, it **MUST** be the latest in date order alphabetically

This removes all the dangerous deleteAll and removeAll mutations from the API. These were only ever used during testing, and aren't used that often anymore in tests anyway.

Tearing down and recreating a Lagoon locally is much easier now than it used to be.

### CODE REVIEW

File: services/keycloak/startup-scripts/00-configure-lagoon.sh [hunk 1]
```diff
@@ -256,7 +256,7 @@ function migrate_admin_organization_permissions {
 
   echo Configuring Organization admin permissions
 
-  echo Delete existing organization management
+  echo Delete deleteall sshkeys
```
Comment (tobybellwood) [inline services/keycloak/startup-scripts/00-configure-lagoon.sh]: que?

Comment (shreddedbacon) [reply services/keycloak/startup-scripts/00-configure-lagoon.sh]: :eyes:

Comment (shreddedbacon) [reply services/keycloak/startup-scripts/00-configure-lagoon.sh]: yah previous merge conflict miss

**Review by tobybellwood — APPROVED** [review]
Works as expected, upgrades track ok too. LGTM

**Pull request merged**